store/storetest/compliance_store.go

```diff
@@ -54,9 +54,7 @@ func testComplianceStore(t *testing.T, ss store.Store) {
 
 	compliances, _ = ss.Compliance().GetAll(1, 1)
 
-	if len(compliances) != 1 {
-		t.Fatal("should only have returned 1")
-	}
+	require.Equal(t, 1, len(compliances), "should only have returned 1")
```

Comment:
You can use `require.Len` here instead of `require.Equal` to improve readability